mLRS/Common/hal/esp/rx-hal-bayckrc-900-pa-esp8285.h

```diff
@@ -2,106 +2,94 @@
 // Copyright (c) MLRS project
 // GPL3
 // https://www.gnu.org/licenses/gpl-3.0.de.html
-// OlliW @ www.olliw.eu
 //*******************************************************
 // hal
 //********************************************************
 
 //-------------------------------------------------------
-// BAYCKRC 900 PA RX
+// ESP8285, ELRS BAYCKRC 900 PA RX
 //-------------------------------------------------------
 
 #define DEVICE_HAS_SINGLE_LED
 #define DEVICE_HAS_NO_DEBUG
 //#define DEVICE_HAS_SERIAL_OR_DEBUG
 
-//-- Timers, Timing, EEPROM, and such stuff
 
-#define DELAY_USE_DWT
+//-- Timers, Timing, EEPROM, and such stuff
 
 #define SYSTICK_TIMESTEP          1000
 #define SYSTICK_DELAY_MS(x)       (uint16_t)(((uint32_t)(x)*(uint32_t)1000)/SYSTICK_TIMESTEP)
 
-#define EE_START_PAGE             0 // 128 kB flash, 2 kB page
+#define EE_START_PAGE             0
 
-#define MICROS_TIMx               TIM15
 
 //-- UARTS
 // UARTB = serial port
 // UART = output port, SBus or whatever
 // UARTC = debug port
 
 #define UARTB_USE_SERIAL
-#define UARTB_BAUD                  RX_SERIAL_BAUDRATE
-#define UARTB_USE_TX
-#define UARTB_TXBUFSIZE             RX_SERIAL_TXBUFSIZE // 1024 // 512
-#define UARTB_USE_TX_ISR
-#define UARTB_USE_RX
-#define UARTB_RXBUFSIZE             RX_SERIAL_RXBUFSIZE // 1024 // 512
+#define UARTB_BAUD                RX_SERIAL_BAUDRATE
+#define UARTB_TXBUFSIZE           RX_SERIAL_TXBUFSIZE
+#define UARTB_RXBUFSIZE           RX_SERIAL_RXBUFSIZE
 
 #define UARTC_USE_SERIAL
-#define UARTC_BAUD                  115200
+#define UARTC_BAUD                115200
 
 
 //-- SX1: SX12xx & SPI
-#define SPI_CS_IO                 15
-#define SPI_FREQUENCY             10000000L
-#define SX_RESET                  2
-#define SX_DIO0                   4
-#define SX_DIO1                   5
+#define SPI_CS_IO                 IO_P15
+#define SPI_FREQUENCY             16000000L
```

Comment:
Friendly note - SX127x SPI should run at 10 MHz.  Only SX128x is rated for 16 MHz.

Reply:
THX! done.